Source/FieldSolver/FiniteDifferenceSolver/MacroscopicProperties/MacroscopicProperties.cpp

```diff
@@ -100,11 +100,11 @@ MacroscopicProperties::InitializeMacroMultiFabUsingParser (
     auto& warpx = WarpX::GetInstance();
     const auto dx_lev = warpx.Geom(lev).CellSizeArray();
     const RealBox& real_box = warpx.Geom(lev).ProbDomain();
+    IntVect iv = macro_mf->ixType().toIntVect() + amrex::IntVect(3);
```

Comment:
instead of amrex::IntVect(3) could you ask how many ghost cells are in macro_mf?
amrex::IntVect(macro_mf->nGrow())

Reply:
will do!